### PR TITLE
Create real-time-project.yml

### PR DESCRIPTION
**Pull request by wingerdkm1**
Someone just introduced our team to Up for Grabs.

Our request will be a great addition

### CODE REVIEW

**Review by wingerdkm1 — COMMENTED** [review]
Not sure why it says line 5 is missing "-"

File: _data/projects/real-time-project.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: Real Time Communications World
+desc: Maintain API Security
+site: https://www.realtimecommunicationsworld.com
+tags:
+  - api
+  - ddos
+  - json
+  upforgrabs:
```
Comment (shiftkey) [inline _data/projects/real-time-project.yml:8]: I think the indents on this line and the next two need to be removed, as our parser expects an array for `tags` and these lines are being parsed as part of it.

**Comment by ritwik12** [comment]
@wingerdkm1 Ping

**Comment by ritwik12** [comment]
@shiftkey Did we stopped automatically closing PRs which stayed more than two weeks? I guess we used to mark them stale

**Comment by shiftkey** [comment]
@ritwik12 I think #1706 was the problem. #1873 should get us back to a good place.

**Comment by ritwik12** [comment]
@shiftkey Got it, Thanks!!

**Pull request closed without merging**